src/python/sourmash_plugin_directsketch/__init__.py

```diff
@@ -176,6 +174,11 @@ def main(self, args):
 
         num_threads = set_thread_pool(args.cores)
 
+        if args.failed is None:
+            args.failed = os.path.basename(args.input_csv) + '.fail.txt'
+        if args.checksum_fail is None:
+            args.checksum_fail = os.path.basename(args.input_csv) + '.ckechsum_fail.txt'
```

Comment:
I think the file extensions should be .csv
```suggestion
            args.failed = os.path.basename(args.input_csv) + '.fail.csv'
        if args.checksum_fail is None:
            args.checksum_fail = os.path.basename(args.input_csv) + '.ckechsum_csv.txt'
```